salt/orch/update.sls

```diff
@@ -0,0 +1,121 @@
+# Generic Updates
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
```

Comment:
The TX update systemd timer will change this flag after the machine has booted and salt has been started. When we notice a machine is out of date, and start this play, we should explicitly refresh all machines so we have as up-to-date info as possible!